### ISSUE TITLE
Already opened note panes should be re-used when navigating to note

### ISSUE THREAD

**Issue by reallyely** [deathau/sliding-panes-obsidian#4]
Great plugin so far, thank you for your work! Here's a feature request you possibly already have in the back of your head. I may try to create a fork to handle this myself, but wanted to log it here first.

In Andy's notes, he re-uses note panes and focuses on them instead of creating a new note to the right
![andy2](https://user-images.githubusercontent.com/1640588/98136608-1f58c500-1e8f-11eb-953b-5715e0cc1664.gif)

Currently this plugin creates a new pane to the right.
![andy](https://user-images.githubusercontent.com/1640588/98136538-0c45f500-1e8f-11eb-8b83-a1660f998a35.gif)

Bonus points for highlighting links in notes when that linked note is are already opened.

**Comment by taylorhadden** [comment]
While I think there should *definitely* be an option for this kind of behavior, I find the current behavior useful as well. As somebody who navigates almost exclusively with the keyboard, I think the ideal would be a shortcut for the current behavior and a shortcut for the proposed behavior, maybe with a settings toggle that can switch the behavior so that clicking works as expected.

**Comment by deathau** [comment]
I agree with both of you: There should be an option to do this, at least 👍

**Comment by deathau** [comment]
Note to self: I might need to resolve #16 first, as I need a way to track file opens when the newly opened pane is not active.

I'll potentially also have to "navigate back" if a file is opened in an active pane (i.e. file opens, if pane exists with that file, go back and then make that other pane active)

**Comment by erichalldev** [comment]
It's worth pointing out that this works well in Andy's notes for a few reasons. You can't move rearrange notes in the stack, and all notes follow the graph left to right. So when you open a new link from a note in the middle of the stack, every note to the right is closed and the new note is opened in their place. The benefit here of reusing an already opened note is that you can't create an infinite loop of notes in the stack. This makes for an intuitive experience of going "down the rabbit hole" without getting lost. 

This might be worth pursuing for this plugin. Maybe it could be an option like "True Andy Mode" or something like that.

**Comment by qubist** [comment]
@erichalldev When first looking at this plugin, _True Andy Mode_ is what I really wanted. It's awesome to have the sliding UI in Obsidian but for me the most important part is what you described about viewing a certain path along the graph. Having the structure of links embedded in the way I interact with the notes really helped me understand the idea of Evergreen notes while reading through Andy's notes, especially the concept that [Evergreen note titles are like APIs](https://notes.andymatuschak.org/z3XP5GRmd9z1D2qCE7pxUvbeSVeQuMiqz9x1C).

I'd be delighted to help develop a _True Andy Mode_ if it's being worked on.
